### PR TITLE
Add Biometric Consent Information

### PR DESCRIPTION
**Pull request by jumaallan**
Story: https://app.shortcut.com/smileid/story/xxx

## Summary

A few sentences/bullet points about the changes

## Known Issues

Any shortcomings in your work. This may include corner cases not correctly handled or issues related
to but not within the scope of your PR. Design compromises should be discussed here if they were not
already discussed above.

## Test Instructions

Concise test instructions on how to verify that your feature works as intended. This should include
changes to the development environment (if applicable) and all commands needed to run your work.

## Screenshot

If applicable (e.g. UI changes), add screenshots to help explain your work.

### CODE REVIEW

File: Example/SmileID/BiometricKYC/BiometricKycWithIdInputScreenViewModel.swift [hunk 1]
```diff
@@ -86,10 +86,17 @@ class BiometricKycWithIdInputScreenViewModel: ObservableObject {
                         )
                     }
                 } else {
-                    // We don't need consent. Proceed forward as if consent has already been granted
+                    // We don't need consent. Mark it as false for this product since it's not needed, unless we want to change this
+                    let consentInfo = ConsentInformation(
+                        consentGrantedDate: ISO8601DateFormatter().string(from: Date()),
```
[the remark this answers is not in the document]
Comment (jumaallan) [reply Example/SmileID/BiometricKYC/BiometricKycWithIdInputScreenViewModel.swift:91]: I will add this on Monday

**Comment by jumaallan** [comment]
> Looks good so far @jumaallan . Adding it to the network request is still missing? Added a comment for the timestamp.

Yes, that is why I left it in draft for now

File: Example/SmileID/BiometricKYC/IdInfoInputScreen.swift [hunk 2]
```diff
@@ -118,9 +120,15 @@ private struct IdInfoInputScreen_Previews: PreviewProvider {
         IdInfoInputScreen(
             selectedCountry: "US",
             selectedIdType: "Driver's License",
+            consentInformation: ConsentInformation(
+                consentGrantedDate: ISO8601DateFormatter().string(from: Date()),
```
[the remark this answers is not in the document]
Comment (jumaallan) [reply Example/SmileID/BiometricKYC/IdInfoInputScreen.swift:124]: thank you

File: Example/SmileID/BiometricKYC/BiometricKycWithIdInputScreenViewModel.swift (hunk 1 above)
[the remark this answers is not in the document]
Comment (jumaallan) [reply Example/SmileID/BiometricKYC/BiometricKycWithIdInputScreenViewModel.swift:91]: I can create it as a util then re use it actually

[the remark this answers is not in the document]
Comment (tobitech) [reply Example/SmileID/BiometricKYC/BiometricKycWithIdInputScreenViewModel.swift:91]: Yes, creating a DateFormatter() is an expensive operation in iOS. It's better to create a reusable that is only initialised once.

[the remark this answers is not in the document]
Comment (jumaallan) [reply Example/SmileID/BiometricKYC/BiometricKycWithIdInputScreenViewModel.swift:91]: I see Japhet has this PR - https://github.com/smileidentity/ios/pull/290

@tobitech @robin-ankele

**Comment by jumaallan** [comment]
> File
> 
> @jumaallan I forgot to mention that the ui seems to be a bit cut off on the consent screen. Is this something we can fix with this PR as well or should we create a separate PR for this?

Sure, I will add some padding too

**Comment by jumaallan** [comment]
Needs #290
